### ISSUE TITLE
Feature Request: Disable update check, when compiling for distributions

### ISSUE THREAD

**Issue by solbu** [flaviotordini/minitube#32]
Hi.
Some distributions distribute Minitube in their package repositories. Mageia does this.
It would be beneficial for distributions to have a compile option that would completely disable update checking, since users normaly can't update packages by other merans that installing a new RPM package for example, as the case is in Mageia.

**Comment by flaviotordini** [comment]
Yes, it would be useful so packagers don't have to maintain a patch for it.
